samples/client/echo_api/typescript-axios/build/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "target": "ES5",
     "module": "commonjs",
     "noImplicitAny": true,
+    "downlevelIteration": true,
```

Comment:
can you rewrite the code so it does not need this change? some folks might be using the generated code without the generated tsconfig.json

Reply:
Yep. I'll use forEach instead. Shouldn't expose any details about iterators.

Reply:
I've made the changes and updated the documentation.

Reply:
And now I've rebased from master again.